channeldb/migration/lnwire21/msat.go

```diff
@@ -55,8 +55,10 @@ func (m MilliSatoshi) String() string {
 // Record returns a TLV record that can be used to encode/decode a MilliSatoshi
 // to/from a TLV stream.
 func (m *MilliSatoshi) Record() tlv.Record {
+	msat := uint64(*m)
```

Comment:
I wonder why we used a pointer in `tlv.SizeBigSize` ?

Reply:
if you mean for the latest diff: i think just to keep other call-sites the same as is today. 

if you mean in general - i think the main call-site was initially just `MakeBigSizeRecord` - which takes pointers so that it can unmarshal into the given types - and so the original function signature of `SizeBigSize` was probably just made to match that so that the code is easy from the perspective of `MakeBigSizeRecord`. But yeah - in future we can probs change that